### PR TITLE
Add build script alias to generate and open API documentation

### PR DESCRIPTION
**Pull request by Hermann-Core**

### CODE REVIEW

**Review by IngridPuppet — COMMENTED** [review]
Is the name of this PR even consistent with these changes? It looks more like you're fixing the workspace configuration than providing a script of any kind.

**Comment by Hermann-Core** [comment]
> Is the name of this PR even consistent with these changes? It looks more like you're fixing the workspace configuration than providing a script of any kind.

The build script alias is defined in **.cargo/config.toml**. Now, by simply typing `cargo docs` in the terminal, we can generate the entire workspace API documentation and open it. Updates to the workspace manifest have been made to reflect these changes.

**Review by IngridPuppet — CHANGES_REQUESTED** [review]
My apologies for my previous comment. I read `OpenAPI`, which you agree, is something completely else 😂😂😂

Please, consider my comments below.

File: Cargo.toml [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
 [workspace]
 members = [
-    "did-endpoint", "generic-server", "mediator-coordination", "mediator-server", "oob-messages", "server-plugin",
+    "did-endpoint",
+    "generic-server",
+    "mediator-coordination",
+    "mediator-server",
+    "oob-messages",
+    "server-plugin"
 ]
```
[the remark this answers is not in the document]
Comment (Hermann-Core) [reply Cargo.toml:14]: > Is it intentional to leave out did-utils?

No it was not intentional. Thanks for addressing that.
> Also, update the crate name of mediator-server so we can have the same in the generated docs, and not didcomm-mediator-rs2.

I noticed that too. Maybe Mr. Francis had a reason for doing that? By the way, I think it’s a good idea for the crates’ names to match the folders’ names, so we don’t have to look at the manifest file to use the correct name.

File: Cargo.toml
[the remark this answers is not in the document]
Comment (Hermann-Core) [reply Cargo.toml:1]: A separate ticket has already been opened to improve the documentation of the overall project. I’m already working on it.

**Review by IngridPuppet — APPROVED** [review]

**Pull request merged**